### ISSUE TITLE
Documentation - basic definitions

### ISSUE THREAD

**Issue by papyr** [skoruba/IdentityServer4.Admin#164]
Many of the features and terms are not well known, preventing easy adoption of a great framework. 

Would be nice to have someone who knows what those terms mean, put the definition in.

**Comment by skoruba** [comment]
Can you please specify what features do you mean? I will add then into docs. 
Thank you

**Comment by xmichaelx** [comment]
Yeah, @skoruba @TomasHubelbauer  are doing this currently in our free time, we want to provide the best value possible but we need some help with docs - well defined requirements on what tutorials/structure of documentation is desired are especially welcome :)

**Comment by papyr** [comment]
Any update on this
